constans/constans.py

```diff
@@ -61,3 +61,42 @@
 # Users
 NAME_USER_1 = 'name1'
 NAME_USER_2 = 'name2'
+
+# constants_scores
+CORRECT_MOVE = "CORRECT_MOVE"
+INVALID_MOVE = "INVALID_MOVE"
+KILL = "KILL"
+ARROW_MISS = "ARROW_MISS"
+GET_ITEMS = "GET_ITEMS"
+TIMEOUT_SC = "TIMEOUT"
+DEATH = "DEATH"
+
+SCORES = {
+    CORRECT_MOVE: 1000,
+    INVALID_MOVE: -100,
+    "GOLD": 10000,
```

Comment:
Should this be a string?

Reply:
Maybe you could put this `SCORES`  below the definition of the `GOLD` and `DIAMOND`  contants (line 93 of this file)

Reply:
Working on it now.